07 visualization callback.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visualization Callback Example\n",
+    "\n",
+    "\n",
+    "## Preamble\n",
+    "Before we dive into creating a callback, we'll need a simple model to work with.  This tutorial uses a model similar to the one in neon's `examples/mnist_mlp.py`, but the same callback should apply to any model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from neon.backends import gen_backend\n",
+    "from neon.data import load_mnist\n",
+    "from neon.data import ArrayIterator\n",
+    "from neon.initializers import Gaussian\n",
+    "from neon.layers import Affine\n",
+    "from neon.transforms import Rectlin, Softmax\n",
+    "from neon.models import Model\n",
+    "from neon.layers import GeneralizedCost\n",
+    "from neon.transforms import CrossEntropyMulti\n",
+    "from neon.optimizers import GradientDescentMomentum\n",
+    "\n",
+    "be = gen_backend(batch_size=128)\n",
+    "(X_train, y_train), (X_test, y_test), nclass = load_mnist(path='stockdata')\n",
```

Comment:
Since this is a publicly available neon course, we should not assume an AMI i think? We probably have to replace some of the other notebook data assumptions to pull from s3 if the data is not available.